lib/src/dart_download.dart

```diff
@@ -0,0 +1,250 @@
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io';
```

Comment:
```suggestion
import 'dart:io' as io;
```

It's easy to introduce some dependency on platform-specific stuff with Dart IO. I like naming it `io` so it stands out :)